### PR TITLE
Remove code duplication for 6.2 handling

### PR DESCRIPTION
**Pull request by IanDelMar**
Alternative to #83 that moves the version sniffing from generate.sh to functionMap.php to remove the code duplication for 6.2 handling.

### CODE REVIEW

**Review by johnbillion — APPROVED** [review]
Let's do it 👍

File: finder.php [hunk 1]
```diff
@@ -41,6 +41,6 @@
     //->notPath('wp-includes/theme-compat/footer.php')
     //->notPath('wp-includes/theme-compat/header.php')
     //->notPath('wp-includes/theme-compat/sidebar.php')
-//    ->notPath('wp-includes/Requests/src')
+    //->notPath('wp-includes/Requests/src')
```
Comment (szepeviktor) [inline finder.php:44]:
```suggestion
```

This bad indentation signified a plan to exclude Request v2 from WC stubs.

File: functionMap.php [hunk 2]
```diff
@@ -1,6 +1,11 @@
 <?php
 
-$httpReturnType = 'array{headers: \Requests_Utility_CaseInsensitiveDictionary, body: string, response: array{code: int,message: string}, cookies: array<int, \WP_HTTP_Cookie>, filename: string|null, http_response: \WP_HTTP_Requests_Response}|\WP_Error';
+$caseInsensitiveDictionary = '\WpOrg\Requests\Utility\CaseInsensitiveDictionary';
+if (file_exists(__DIR__ . '/source/wordpress/wp-includes/Requests/Cookie/Jar.php')) {
+    $caseInsensitiveDictionary = '\Requests_Utility_CaseInsensitiveDictionary';
+}
+
+$httpReturnType = "array{headers: $caseInsensitiveDictionary, body: string, response: array{code: int,message: string}, cookies: array<int, \WP_HTTP_Cookie>, filename: string|null, http_response: \WP_HTTP_Requests_Response}|\WP_Error";
```
Comment (szepeviktor) [inline functionMap.php:8]: Low tech please.



```suggestion
$httpReturnType = 'array{headers: \WpOrg\Requests\Utility\CaseInsensitiveDictionary, body: string, response: array{code: int,message: string}, cookies: array<int, \WP_HTTP_Cookie>, filename: string|null, http_response: \WP_HTTP_Requests_Response}|\WP_Error';

if (file_exists(__DIR__ . '/source/wordpress/wp-includes/Requests/Cookie/Jar.php')) {
    $httpReturnType = 'array{headers: \Requests_Utility_CaseInsensitiveDictionary, body: string, response: array{code: int,message: string}, cookies: array<int, \WP_HTTP_Cookie>, filename: string|null, http_response: \WP_HTTP_Requests_Response}|\WP_Error';
}
```

Comment (szepeviktor) [reply functionMap.php:8]: This is no enterprise code.

**Comment by szepeviktor** [comment]
Oh no! Merging #82 caused turmoil here.

**Comment by szepeviktor** [comment]
Thank you.

File: finder.php (hunk 1 above)
Comment (IanDelMar) [reply finder.php]: Did you mean the bad indentation should be kept or the whole line should be dropped?

Comment (szepeviktor) [reply finder.php]: Removed! Plan abandoned.

Removed in master.

**Pull request merged**